### ISSUE TITLE
Fappello eror

### ISSUE THREAD

**Issue by KetupatBulat** [Ximaz/megaloader#9]
Traceback (most recent call last):
  File "c:\Users\user\Documents\GitHub\megaloader\test.py", line 12, in <module>
    main()
  File "c:\Users\user\Documents\GitHub\megaloader\test.py", line 8, in main
    for u in api.export():
  File "c:\Users\user\Documents\GitHub\megaloader\plugins\fapello.py", line 41, in export
    for e in self.__get_medias_from_page(l):
  File "c:\Users\user\Documents\GitHub\megaloader\plugins\fapello.py", line 33, in __get_medias_from_page   
    page = requests.get(page_url).text
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\requests\api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\requests\sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\requests\sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\requests\adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

i keep getting this issue when attempting to run the gofile plugin, any idea as to what the issue is?

**Comment by Ximaz** [comment]
Can you post the link you was trying to reach ?

**Comment by KetupatBulat** [comment]
https://fapello.com/mewnii638/

**Comment by Ximaz** [comment]
So, I read something of StackOverflow about that error, it seems it's not part of my code, but it's because Fapello's servers might encountered an issue when time comes to send the response, either because malformated request (which shouldn't be the case because I copied the client requests headers), or because it triggered a crash on their side. I tried to download the same content as you did and it worked fine. I can't help you on this one, just try again.
